### ISSUE TITLE
`Toggle` value not updated when it's reset from a binding

### ISSUE THREAD

**Issue by MaxDesiatov** [TokamakUI/Tokamak#287]
Previously reported by @rbartolome. Here's a reproducible test case:

```swift
class TestData: ObservableObject
{
    @Published var stringData: String = ""
    @Published var booleanData: Bool = false
    
    func reset()
    {
        self.stringData = ""
        self.booleanData = false
    }
}

struct DemoToggle: App
{
    @ObservedObject var testdata = TestData()
    var body: some Scene
    {
        WindowGroup("DemoTracker")
        {
            VStack
            {
                Toggle("Toogle", isOn: self.$testdata.booleanData)
                TextField("Placeholder", text: self.$testdata.stringData)
                Button(action:
                {
                    self.testdata.reset()
                },
                label:
                {
                    Text("Reset")
                })
            }
        }
    }
}

DemoToggle.main()
```

**Comment by RayZhao1998** [comment]
It seems that `checked.toggle()` can't update the Toggle value too.

Full code:

```swift
public struct ToggleDemo: View {
  @State var checked = false

  public var body: some View {
    VStack {
      Toggle("Toogle", isOn: $checked)
      Text(checked ? "Checked" : "Not Checked")
      Button(action:
      {
        self.checked.toggle()
      },
      label:
      {
          Text("Reset")
      })
    }
  }
}
```

**Comment by RayZhao1998** [comment]
Found the reason. The checkbox in HTML is so weird. Whether it's checked is not determined by the checked property. Even though I uncheck the checkbox, the property `checked="checked"` is still there.
<img width="950" alt="截屏2020-11-27 上午1 02 50" src="https://user-images.githubusercontent.com/22616933/100377481-4cdafd80-304c-11eb-8d30-3e206fb7adaf.png">

**Comment by MaxDesiatov** [comment]
Thanks for the investigation! Now I remember something similar was mentioned during review when @j-f1 was working on https://github.com/TokamakUI/Tokamak/pull/159.

**Comment by RayZhao1998** [comment]
Usually we uncheck the checkbox in JS like `getElementsByTagName('input')[0].checked = false`.

I find a modifier: `_domRef`. Can we get the input DOM and set its checked?

**Comment by MaxDesiatov** [comment]
@RayZhao1998 a fix for this was included as a part of the latest 0.5.3 release.
